docs/source/install.md

```diff
@@ -0,0 +1,137 @@
+# Install Fabric Samples
```

Comment:
When I first read through this, I kept looking for "install Fabric", ie how do I get it on my machine. I wonder if we should rename this - Install Fabric and samples (we can omit the binaries since no-one at this point knows that those are)

Reply:
Done.